### DISCUSSION TITLE
add ower to TaskInstance class

### DISCUSSION THREAD

**Discussion by wornjs** [apache/airflow#46010] [Ideas]
### Description

add task owner information to taskinstance

### Use case/motivation

I am sending an alarm to Slack using the airflow callback(on_success_callback, on_failure_callback) function. 

I create an alarm message using the information of the task instance, and it would be better if the task owner information is included here.

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
I think this one is interesting and rather new idea (note that you already have `owner` field available via Context -> task_instance - so I assume you are talking about some "owner" human.

If that's a good guess - it's mostly a discussion and if you would like to continue the discussion, my advice is to be more specific on what you mean by `owner`. How you want to determine the owner? How to get that information, how it behaves in different scenarios, who - from the actors described in our security/user model - https://airflow.apache.org/docs/apache-airflow/stable/security/security_model.html  is the owner - how does it work in different situations where tasks are scheduled, triggered by assets, APIs etc. etc. 

So there are a lot of things to unpack here. And my suggestion is that if you have all those things thought over and good description of your proposal explaining all the scenaris and cases - bring it to the devlist and explain what your proposal is.
